bench_pytorch/bench.sh

```diff
@@ -125,21 +110,20 @@ while [ "$#" -gt 0 ]; do
                 "cuda" | "metal" | "cpu")
                     ;;
                 *)
-                    echo "Invalid value for --device. Please use 'cuda', 'gpu' or 'cpu'."
+                    echo "Invalid value for --device. Please use 'cuda', 'cpu' or 'metal'."
                     print_usage
                     ;;
             esac
             if [ "$DEVICE" == "cuda" ]; then
                 check_cuda
+            else
+                echo "Not supported for $DEVICE"
+                exit 1
             fi
             shift 2
             ;;
-        -lf|--log_file)
-            LOG_FILENAME="$2"
-            shift 2
-            ;;
-        -md|--models_dir)
-            MODELS_DIR="$2"
+        -n|--model_name)
+            MODEL_NAME="$2"
```

Comment:
can we check tha the model belongs to llama or mistral?

Reply:
> can we check tha the model belongs to llama or mistral?

This thing is already been checked inside this line:

https://github.com/premAI-io/benchmarks/blob/13c5ef6edcfd288072ce3836f8027738f3d2c22e/common/base.py#L34-L36